scripts/release-binary.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -p gitAndTools.hub git -i bash
+project=$(basename $(pwd))
+
+TEMPDIR=`mktemp -d`
+
+REV=`git rev-parse HEAD`
+
+nix build -f release -o $TEMPDIR/$project
+
+tar --owner=serokell:1000 --mode='u+rwX' -czhf $TEMPDIR/release.tar.gz -C $TEMPDIR $project
+
+hub release delete auto-release
```

Comment:
According to the documentation it does not delete tag `auto-release`, so AFAIU each release will be created for the same tag. I think you should also delete the tag here.